src/orquestra/sdk/_base/cli/_dorq/_arg_resolvers.py

```diff
@@ -115,18 +118,43 @@ def __init__(
         self._wf_run_repo = wf_run_repo
         self._prompter = prompter
 
+    def _prepare_wf_list_for_prompt(self, wfs):
+        # sort wfs by submission date. Take into account when there is no start_time
+        wfs.sort(
+            key=lambda wf: wf.status.start_time
+            if wf.status.start_time
+            else datetime.datetime.fromtimestamp(0),
+            reverse=True,
+        )
+        # Create labels of wf that are printed by prompter
+        # Label is <wf_id> <start_time> tabulated nicely to create good-looking table
+        labels = [
+            [
+                wf.id,
+                wf.status.start_time.astimezone().replace(tzinfo=None).ctime()
```

Comment:
**suggestion**: I'm assuming this is about generating a human-friendly string with a local time. Datetimes in Python are footguns. WDYT about extracting `.astimezone().replace(tzinfo=None).ctime()` to a separate function to state the purpose explicitly?